README.md

```
@@ -31,17 +52,29 @@ In a Claude Code session, run:
 ```bash
 claude mcp add whiteboard -- npx -y @kamiazya/whiteboard-mcp@latest
 ```
 
-### Codex CLI
+</details>
+
+### Codex
+
+In a Codex session, run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)